checkov/common/util/igraph_serialization.py

```diff
@@ -7,7 +8,7 @@
     from igraph import Graph
 
 
-def serialize_to_json(igraph: Graph) -> Dict[str, Any]:
+def serialize_to_json(igraph: Graph, absolute_root_folder: str) -> Dict[str, Any]:
```

Comment:
can you make the new parameter optional? we copied the serializer here and will need to remove it from the platform and use this function instead. In the other cases, we won't save the `absolute_root_folder`

Reply:
fixed